src/tools/genpolicy/src/mount_and_storage.rs

```diff
@@ -301,8 +301,9 @@ fn get_shared_bind_mount(
     propagation: &str,
     access: &str,
 ) {
-    let mount_path = if let Some(byte_index) = str::rfind(&yaml_mount.mountPath, '/') {
-        str::from_utf8(&yaml_mount.mountPath.as_bytes()[byte_index + 1..]).unwrap()
+    let path = Path::new(&yaml_mount.mountPath);
+    let mount_path = if let Some(file_name) = path.file_name() {
```

Comment:
According to [the docs](https://doc.rust-lang.org/std/path/struct.Path.html#method.file_name), `file_name` returns `None` on paths such as `/` and `/foo/bar/...`. Is it correct to default to the full `mountPath` in these cases? It would be helpful to point to the host-side implementation of this.

Reply:
@burgerdev , good points, thanks!

- Added the host-side implementation reference.
- Replaced the default path with unwrap(). 

Path::file_name() seems similar enough to go's filepath.Base() for commonly used mount paths. Unusual mount paths might get rejected by genpolicy here.